### ISSUE TITLE
Add information about delay on "rbenv install 2.5.1" step

### ISSUE THREAD

**Issue by polnetwork** [Platoniq/decidim-install#20]
The setup is explained very clear, the only point where I'v doubt is on the "rbenv install 2.5.1" command because this could take a while and there's no visual information about the progress. Maybe we can explain on the documentation that.
Thank's for all your work.

**Comment by microstudi** [comment]
Good Idea, I'll incorporate that. I need to follow the guide myself to be sure thinks are accurate!
